src/test/java/de/filefighter/rest/cucumber/CommonCucumberSteps.java

```diff
@@ -110,12 +110,23 @@ public void fileSystemItemWithTheFileSystemIdExistsAndHasTheName(long fileSystem
     @And("fileSystemItem with the fileSystemId {long} is a folder and contains the fileSystemId {long}")
     public void fileSystemItemWithTheFileSystemIdIsAFolderAndContainsTheFileSystemId(long fileSystemIdFolder, long fileSystemId) {
         Query query = new Query();
-        Update newUpdate = new Update().set("itemIds", new long[]{fileSystemId});
+        Update newUpdate = new Update()
+                .set("itemIds", new long[]{fileSystemId})
```

Comment:
does this update or overwrite? It should update, as written in the feature file.

Reply:
currently it does overwrite it. you want me to add this id to the array instead?

Reply:
yes, otherwise you can't create folders with multiple files in them

Reply:
kk